server/di-container.ts

```diff
@@ -43,6 +45,16 @@ export function configureContainer() {
       validateReply: awilix.asValue(validateReply),
       validateReplyPUT: awilix.asValue(validateReplyPUT),
       logger: awilix.asClass(Logger),
+      Env: awilix.asValue(
+        plainToClass(Environment, {
```

Comment:
This thing is now done twice. Rather when validating env if it succeeds return the valid env class other wise throw error. Then `configureContainer` can have then validated env as parameter 👍